### ISSUE TITLE
after update and option to remove GoFish5, still get button with GoFish5 on startup

### ISSUE THREAD

**Issue by matthewaceware** [VFPX/GoFish#144]
## ℹ Computer information

- VFP version: 09.00.0000.5815
- GoFish version: 6.2.007
- Last GoFish version w/o bug (if available):

## 📝 Provide detailed reproduction steps (if any)

1. Launch Thor
2. I must have set up a button for GoFish5 previously but I am still getting it and am unsure how to even turn it off
3. The button does nothing since I've removed GoFish5 in the last update

### ✔️ Expected result

The button should be going to GoFish. Though, at the very least, I need to figure out how to not have that button show up.

### ❌ Actual result

Nothing.

## 📷 Screenshots

![image](https://github.com/VFPX/GoFish/assets/31904139/8ffc1752-abab-4f4c-a154-bf7ecaeff6ed)

Thanks,
Matthew

**Comment by lscheffler** [comment]
GoFish5 create files in the Thor structure. Even if not installed via Thor.   
The best solution would be to search the Thor directory with GoFish or Code Reference for the menu string, and delete the file.

**Comment by Jimrnelson** [comment]
Matthew --

Where are you seeing this image?  Please supply some context ...

As it happens, this is one of those murky areas where it's difficult to identify where the problem is.  I believe that this more likely a Thor problem than one from GF.

**Comment by Jimrnelson** [comment]
As for where this goes, Thor or GF, no guess yet.

That actually looks like a tiny form, doesn't it, not a button?

Does it stay there when you get to the command window?  Can you use Object Explorer to look at it, find out what class and class library it belongs to?

If not, can you get the same information from _Screen.Forms[nn] -- no telling what nn is for you, but try numbers starting with one.

**Comment by matthewaceware** [comment]
Yes, it does look like a tiny form. I usually drag this onto the toolbar so it is a button on the toolbar: 
![image](https://github.com/VFPX/GoFish/assets/31904139/52bc92c9-33a4-42e0-ba3e-76570f7c533b)

I can't seem to get to it from the Object Explorer or using the _Screen.Forms (since it isn't a form and is instead a toolbar).

Thanks,
Matthew
